### PR TITLE
🧹 Chore/67 unit tests

### LINKED ISSUE #67: 🧹 Unit Tests
## Description
The RedwoodJS scaffold generator doesn’t generate stories or tests for any of its generated things except the api services. As such the following unit tests and stories are missing from the template application:
- Pages:
  - Admin (Role/Team/Users) CRUD
  - Authentication (Forgot Password/Login/NotFound/Reset Password/Sign Up)
  - Fatal Error
- Layouts:
  - Admin (Role/Team/Users)
  - Main
- Components:
  - Admin (Role/Team/Users) CRUD
  - ZealLogo
## Affects
- Unit Testing
## Consequences
Won’t have testing coverage of code. Established poor patterns.

### PR DESCRIPTION
**Pull request by nataliadiaz2**
# Pull Request

## Story

Fix #67 Chore/67 unit tests

## Description

Web side tests for layouts and pages to test if pages are rendering

## Solution

Add tests to web side components and pages previously without to test rendering

## Affected Areas

Admin Layouts, Main Layout, Forgot Password/Login/NotFound/Sign Up Pages

## Tests

Test for Layouts and necessary Pages for rendering. 

## Was this feature tested on all browsers?

- [X] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Internet Explorer
- [ ] Safari

## Definition of Done

- [ ] README updated
- [X] Tests written
- [ ] Acceptance criteria implement
- [ ] Code reviewed
- [ ] Feature accepted

## PR Comments Emoji Legend

😻 - `:heart_cat_eyes:` Compliment to code/idea/etc
♻️ - `:recycle:` Non-blocking proposed refactor
➕ - `:heavy_plus_sign:` Non-blocking proposed addition
🔥 - `:fire:` Non-blocking proposed removal
❓ - `:question:` Non-blocking question
⚠️ - `:warning:` Blocking comment that must be addressed before PR

## Gifs for life (required)

mandatory

### CODE REVIEW

File: web/src/Routes.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { Set, Router, Route, Private } from '@redwoodjs/router'
 
-import { AdminLayout } from './layouts/Admin/Layout'
+import { AdminLayout } from './layouts/Admin/AdminLayout/Layout'
```
Comment (craineum) [inline web/src/Routes.tsx:3]: ❓ Why did you rename/move these layouts?

File: web/src/pages/FatalErrorPage/FatalErrorPage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { render } from '@redwoodjs/testing/web'
+
+import RedwoodDevFatalErrorPage from './FatalErrorPage'
```
Comment (craineum) [inline web/src/pages/FatalErrorPage/FatalErrorPage.test.tsx:3]: ❓ Why prepend with `RedwoodDev`?

File: web/src/pages/ResetPasswordPage/ResetPasswordPage.tsx [hunk 3]
```diff
@@ -60,7 +60,7 @@ const ResetPasswordPage = ({ resetToken }) => {
     <>
       <MetaTags title="Reset Password" />
 
-      <main className="rw-main">
+      <main className="rw-main" data-testid="resetpassword-page">
```
Comment (craineum) [inline web/src/pages/ResetPasswordPage/ResetPasswordPage.tsx:63]: 🔥 This feels like it is in the wrong PR.

**Review by craineum — COMMENTED** [review]
Couple of questions

File: web/src/pages/FatalErrorPage/FatalErrorPage.test.tsx (hunk 2 above)
Comment (nataliadiaz2) [reply web/src/pages/FatalErrorPage/FatalErrorPage.test.tsx:3]: I assumed it had to be imported that way to correlate with the export default on the FatalErrorPage, I can update to remove the RedwoodDev.

File: web/src/pages/ResetPasswordPage/ResetPasswordPage.tsx (hunk 3 above)
Comment (nataliadiaz2) [reply web/src/pages/ResetPasswordPage/ResetPasswordPage.tsx:63]: Removed

File: web/src/Routes.tsx (hunk 1 above)
Comment (nataliadiaz2) [reply web/src/Routes.tsx:3]: I combined Layout and LayoutTest to one folder, then moved under Admin to keep admin layouts together.

File: web/src/pages/FatalErrorPage/FatalErrorPage.test.tsx (hunk 2 above)
Comment (craineum) [reply web/src/pages/FatalErrorPage/FatalErrorPage.test.tsx:3]: Looking into it more, there is a RedwoodJS component that is actually drawing this page. So let's not change much here. I am good with it being prepended with `RedwoodDev`.

File: web/src/Routes.tsx (hunk 1 above)
Comment (craineum) [reply web/src/Routes.tsx:3]: ♻️ I think it would be better if this would be? `./layouts/Admin/AdminLayout` without the added directory.

**Review by craineum — APPROVED** [review]
🚀
